tests/v2_tests/test_fs.py

```diff
@@ -126,6 +126,48 @@ def test_factory_open():
         assert isinstance(fs, S3)
 
 
+@mock_s3
+def test_from_url_create_option():
+    # posix filesystem
+    with tempfile.TemporaryDirectory() as d:
+        path = os.path.join(d, 'non-existent-directory')
+        with pytest.raises(ValueError):
+            from_url(path)
+
+        from_url(path, create=True)
+        assert os.path.exists(path) and os.path.isdir(path)
+
+    # HDFS
+    hdfs = Hdfs()
```

Comment:
Any test that uses `Hdfs()` requires annotation to skip it in CI like in `v2_tests/test_hdfs.py` . I'd recommend moving this section to `test_hdfs.py` to make it clearer.
```python
@unittest.skipIf(shutil.which('hdfs') is None, "HDFS client not installed") 
```

Reply:
Thanks for telling me that :bow::bow:

Right, it should be then moved to `test_hdfs.py` and marked as skip (in CI).
Maybe I can slightly reorganize the tests for `open_url` and `from_url` in `test_fs.py` factories so contributors can write more filesystem-specific precise tests (like in #247) in each `test_xxx.py`.